go.sum

```diff
@@ -1,7 +1,11 @@
 chainguard.dev/apko v0.14.8 h1:UHnn4qm/erRppygHH8/0OEA+E72fnwAY2px/YaRaI8g=
 chainguard.dev/apko v0.14.8/go.mod h1:aFEwAkFsf7sXvVFQ2ui6KRK3tbG3mIl5PmPd4JqKGVM=
+chainguard.dev/apko v0.15.0 h1:lbO/dfa602gxuidh1eqaUNwolvrEa73+Fv/du3ByY6s=
+chainguard.dev/apko v0.15.0/go.mod h1:41l++xjHz0zwntVJg/B5r2qWPUxF4lfvvJhkDY2nPgI=
```

Comment:
This was produced without running go mod tidy to update hashes instead of adding duplicate ones.

Please see https://github.com/wolfi-dev/wolfictl/pull/967/files instead.

Reply:
go mod tidy run.